Sources/Storage.swift

```diff
@@ -91,7 +93,15 @@ extension Storage {
             .trueOrThrow(StorageError.createFile)
     }
 
-    func commonLoad<T>(forKey key: String, fromData: (Data) throws -> T) throws -> T {
+    func commonLoad<T>(forKey key: String, withMaxAge maxAge: TimeInterval, fromData: (Data) throws -> T) throws -> T {
+        func verify(maxAge: TimeInterval, forKey key: String) throws -> Bool {
+            Date().timeIntervalSince(try modificationDate(forKey: key)) <= maxAge
```

Comment:
Maybe we can have `Date()` to be a closure on Storage, so we can unit test it

Reply:
I'm not sure what you meant but I hope that's what is on your mind. 20102cd86d4df3d9786a00e0982ef0c078b3e575